packages/flame/lib/src/components/component.dart

```diff
@@ -765,28 +760,28 @@ class _LifecycleManager {
     return !(_children.isEmpty && _removals.isEmpty && _adoption.isEmpty);
   }
 
+  /// Attempt to resolve pending events in all lifecycle event queues.
+  ///
+  /// This method must be periodically invoked by the game engine, in order to
```

Comment:
Periodically -> each tick?

Reply:
Currently we do call this method on every game tick.

But nothing stops us from calling it more or less often. For example, someone might decide to call this twice per tick - once at the beginning of update and once after the render. Or, conversely, you could decide to call this less often if the fps becomes too low.

Seems like "periodically" fits quite well to these.